features/step_definitions/run_steps.rb

```diff
@@ -64,7 +64,12 @@
   expected_commands.map_column! 'COMMAND' do |command|
     command.gsub(sha_regex) do |sha_expression|
       commit_message = sha_expression.match(sha_regex).captures[0].strip
-      output_of "git reflog --grep-reflog='commit: #{commit_message.strip}' --format='%H'"
```

Comment:
an alternative way of going about this would be
```ruby
output_of "git reflog --grep='#{commit_message.strip}' --format='%H' -1"
```
`--grep` searches just the commit message whereas `--grep-reflog` searches the reflog message.
It we search just the commit message there may be multiple results, (checkouts that are on the same commit will also show up), but adding the `-1` limits it to a single result.